### PR TITLE
Update documentation for new spinal levels

### LINKED ISSUE #4247: Update documentation for new spinal levels
Alongside https://github.com/spinalcordtoolbox/PAM50/pull/18, update the documentation to reflect the changes, notably: the pictures, the references to Cadotte et al. and any reference to a "separate" spinal level folder. Also make sure to remove the term "probabilistic" spinal levels.

TODO:

- [x] update figure
- [x] replace paperpile links with open-access links when applicable
- [ ] Fix "14, point-wise spinal level labels, PAM50_label_spinal_levels.nii.gz" in info_label
- [ ] Add correct entry in info_label

### PR DESCRIPTION
**Pull request by jcohenadad**
#### GitHub

- [x] I've given this PR a concise, self-descriptive, and meaningful title
- [x] I've linked relevant issues in the PR body
- [x] I've applied [the relevant labels](https://intranet.neuro.polymtl.ca/geek-tips/contributing#pr-labels-a-href-pr-labels-id-pr-labels-a) to this PR
- [x] I've applied a [release milestone](https://github.com/spinalcordtoolbox/spinalcordtoolbox/milestones) (major, minor, patch) in line with [Semantic Versioning guidelines](https://github.com/spinalcordtoolbox/spinalcordtoolbox/wiki/Misc%3A-Creating-a-new-release#convention-for-naming-releases) 
- [x] I've assigned a reviewer

#### PR contents

- [x] I've consulted [SCT's internal developer documentation](https://github.com/spinalcordtoolbox/spinalcordtoolbox/wiki) to ensure my contribution is in line with any relevant design decisions
- [ ] I've added [relevant tests](https://github.com/spinalcordtoolbox/spinalcordtoolbox/wiki/Testing) for my contribution
- [ ] I've updated the [relevant documentation](https://github.com/spinalcordtoolbox/spinalcordtoolbox/wiki/Documentation) for my changes, including argparse descriptions, docstrings, and ReadTheDocs tutorial pages

## Linked issues
Fixes #4247

### CODE REVIEW

**Review by joshuacwnewton — APPROVED** [review]
This LGTM! Though, we should probably wait to merge until the actual PAM50 release has been updated, because documentation changes go live immediately.

**Review by sandrinebedard — APPROVED** [review]
Looks great!

Would be nicce maybe to have a picture of what it looks like next to it:

image

**Comment by jcohenadad** [comment]
Here's what I'm thinking of using 
spinal_levels

Source: `GDrive/NeuroPoly/Projects/SCT/fig/figures.key`

**Comment by sandrinebedard** [comment]
> Here's what I'm thinking of using spinal_levels
> 
> Source: `GDrive/NeuroPoly/Projects/SCT/fig/figures.key`

Looks great! We don't see that well the nerve rootlet from C3, but I think it's fine!

**Comment by jcohenadad** [comment]
> We don't see that well the nerve rootlet from C3, but I think it's fine!

Yes I agree-- I wanted to run it by everyone in the team to see if someone one react to this issue, and the fact that you noticed means that others will also notice, so I updated the figure:

spinal_levels

File: documentation/source/overview/concepts/pam50.rst [hunk 1]
```diff
@@ -53,33 +72,44 @@ The atlas folder also contains an ``info_label.txt`` file to explain what each f
...
-Spinal levels (``PAM50/spinal_levels``)
-=======================================
+Spinal levels
+=============
 
-.. figure:: https://raw.githubusercontent.com/spinalcordtoolbox/doc-figures/master/pam50/spinal_levels_illustration.png
-    :figwidth: 40%
+**Location:** ``PAM50/template/PAM50_spinal_levels.nii.gz``
+
+.. figure:: https://raw.githubusercontent.com/spinalcordtoolbox/doc-figures/master/pam50/spinal_levels.png
```
Comment (jcohenadad) [inline documentation/source/overview/concepts/pam50.rst:85]: I still find the pink writing a bit too large

**Comment by jcohenadad** [comment]
Sorry, this commit message closed the PR (https://github.com/spinalcordtoolbox/doc-figures/commit/5a9ca65d6c2a9407a47d20bd63189426bf553942)

File: documentation/source/overview/concepts/pam50.rst (hunk 1 above)
Comment (jcohenadad) [reply documentation/source/overview/concepts/pam50.rst:85]: Fixed in https://github.com/spinalcordtoolbox/doc-figures/commit/5a9ca65d6c2a9407a47d20bd63189426bf553942

**Pull request merged**